akvo/rsr/management/commands/link_results_framework_to_parent_project.py

```diff
@@ -16,7 +16,7 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         project = Project.objects.get(id=options['project_id'])
-        parent = project.parents_all().first()
+        parent = project.parent
```

Comment:
`parent()` ?

Reply:
Yes, I think this should be a call to the `.parent()` method.

Reply:
Fixed in 60a49bc68c90630254bbe7a232c53d28427c8d9f